src/Hedgehog/Gen.fs

```diff
@@ -368,13 +355,13 @@ module Gen =
 
     // Generates a random character in the specified range.
     let charRange (lo : char) (hi : char) : Gen<char> =
-        range (int lo) (int hi) |> map char
+        integral <| Range.constant (int lo) (int hi) |> map char
 
     /// Generates a random character.
     let char : Gen<char> =
         let lo = System.Char.MinValue
         let hi = System.Char.MaxValue
-        sizedRange (int lo) (int hi) |> map char
```

Comment:
I'm curious, does `char` work with the `Range` combinators? or is it too different from an integer?
If it does, then we should delete `charRange` and change this to:
```fs
char (range : Range<char>)
```
If we can't use characters with `Range` then we should have:
```fs
char (lo : char) (hi : char)
charBounded : Gen<char>
```
to match the `enum` / `enumBounded` combinators in Haskell.